lms/djangoapps/django_comment_client/utils.py

```diff
@@ -124,9 +124,9 @@ def has_required_keys(module):
     return True
 
 
-def get_accessible_discussion_modules(course, user, include_all=False):  # pylint: disable=invalid-name
+def get_accessible_discussion_xblocks(course, user, include_all=False):  # pylint: disable=invalid-name
     """
-    Return a list of all valid discussion modules in this course that
+    Return a list of all valid discussion xblocks in this course that
```

Comment:
In this function, as well as the next, we continue to use the word `module` in many places.  In the file above, we use the phrase `discussion module` in some comments where we are describing a change with this rename.

I'm clear that searching for "discussion module" and replacing with "discussion xblock" would be useful.  I'm not clear on how important it would be to do other renaming. 
@ormsbee: Do you have an opinion on this?